### ISSUE TITLE
[TIL] 문승태 - 201021

### ISSUE THREAD

**Issue by seungtaemoon** [codestates/ds-TIL#632]
<h1> Month 2 - 여정의 시작 </h1>
# Confusion Matrix

<h3> 오늘 배운 것: </h3>

1. Confusion Matrix:
- True Positive
- True Negative
- False Positive
- False Negative
- 정확성(Accuracy)
- 정밀도(Precision)
- 재현율(Recall)
- Beta
- 분류모델의 평가지표(F1_score)

<h3> 앞으로의 과제: </h3>
- 어떻게 해야 TP, TN을 최적화 할 수 있는지 조금 더 고민할 필요가 있다.
- 분류 문제에서는 F1 Score가 중요하다.
- 상황에 따라 각 지표중 중점을 두어야 할 것이 다르다고 느꼈다.

<h3> 그 외 느낀 점: </h3>

- Sklearn이나 Scikit-Learn의 머신 러닝 기능들을 잘 이해하기 위해서는, Doc을 읽어보아야 할 것 같다.
- True Positive와 True Negative를 잘 구분하게 하는 것이 매우 중요하다는 것을 깨달았다.
- 분류 문제에 있어서는 F1 Score가 중요하다. 이에 대해서 좀 더 읽어보고 알아봐야겠다.
- 상황에 따라 정확도, 재현율, 정밀도 등의 중요성이 다를 수 있다는 것에 대해 감을 잡을 필요가 있다.
